meta-xt-driver-domain/recipes-extended/displaymanager/displaymanager_git.bb

```diff
@@ -10,10 +10,9 @@ SYSTEMD_SERVICE_${PN} = "display-manager.service"
 DEPENDS = "libconfig wayland-ivi-extension glib-2.0 glib-2.0-native git-native xt-log"
 
 LIC_FILES_CHKSUM = "file://LICENSE;md5=b234ee4d69f5fce4486a80fdaf4a4263"
-RDEPENDS_${PN} += " dbus-cxx"
```

Comment:
This is not covered in the commit message. You can reword it like this:

> displaymanager: drop dbus-cxx dependency
> 
> We can't use dbus-cxx because it is under GPLv3 license. Switch
> displaymanager to 'master' branch, which does not depend on dbus-cxx
> and remove corresponding RDEPEND

Reply:
fixed